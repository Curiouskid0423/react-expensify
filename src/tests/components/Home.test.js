import React from "react";
import ExpenseDashboard from "../../components/Home";
import { shallow } from "enzyme";

describe("ExpenseDashboard Component", () => {
    it("Should render a Dashboard object w/o knowing how " +
        "nested components have changed.", () => {
       const wrapper = shallow(<ExpenseDashboard />);
       expect(wrapper).toMatchSnapshot();
    });
})