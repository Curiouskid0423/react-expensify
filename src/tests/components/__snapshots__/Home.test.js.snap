// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseDashboard Component Should render a Dashboard object w/o knowing how nested components have changed. 1`] = `
<div>
  <h3>
     - MAIN DASHBOARD 
  </h3>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
